src/openzaak/components/zaken/tests/test_zaakinformatieobjecten.py

```diff
@@ -394,7 +396,359 @@ def test_delete(self):
         self.assertTrue(Zaak.objects.exists())
 
 
+@override_settings(CMIS_ENABLED=True)
```

Comment:
Can you move the classes with `CMIS_ENABLED=True` tests to a file with the postfix `_cmis`, so in this case `test_zaakinformatieobjecten_cmis.py`?